### PR TITLE
Update as preparation for scala 12

### PR DESCRIPTION
**Pull request by alessandrosimi-sa**

### CODE REVIEW

File: src/test/scala/com/redis/Bench.scala [hunk 1]
```diff
@@ -31,8 +35,8 @@ object Bench {
 
   def load(opsPerClient: Int, fn: (Int, String) => Unit)(implicit clients: RedisClientPool): (Double, Double, Seq[_]) = {
     val start = System.nanoTime
-    val tasks = (1 to 100) map (i => actors.Futures.future { fn(opsPerClient, "k" + i.toString) })
-    val results = tasks map (future => future.apply())
+    val tasks = (1 to 100) map (i => Future { fn(opsPerClient, "k" + i.toString) })
+    val results = Await.result(Future.sequence(tasks), Duration.Inf)
```
Comment (shinyhappydan) [inline src/test/scala/com/redis/Bench.scala:39]: This might run forever unless you specify a timeout

**Review by ncorreia — APPROVED** [review]
lgtm

File: src/test/scala/com/redis/cluster/RedisShardsSpec.scala [hunk 2]
```diff
@@ -23,9 +23,21 @@ with BeforeAndAfterAll {
...
+      //throw t
+  }
 
-  override def afterAll = r.close
+  override def afterAll = try {
+    r.close
+  } catch {
+    case t: Throwable =>
+      t.printStackTrace()
+      //throw t
```
Comment (shinyhappydan) [inline src/test/scala/com/redis/cluster/RedisShardsSpec.scala]: why not throw this..?

File: pom.xml [hunk 3]
```diff
@@ -50,11 +50,6 @@
 			<version>${scalalib.version}</version>
 		</dependency>
 		<dependency>
-			<groupId>org.scala-lang</groupId>
-			<artifactId>scala-actors</artifactId>
-			<version>${scalalib.version}</version>
-		</dependency>
```
Comment (ncorreia) [inline pom.xml:56]: Nice one :-)

File: src/test/scala/com/redis/Bench.scala (hunk 1 above)
Comment (alessandrosimi-sa) [reply src/test/scala/com/redis/Bench.scala:39]: Good point, I need to check the actor.Future implementation to see what future.apply() does :(

File: src/test/scala/com/redis/cluster/RedisShardsSpec.scala (hunk 2 above)
Comment (alessandrosimi-sa) [reply src/test/scala/com/redis/cluster/RedisShardsSpec.scala]: Some tests fails on my machine, not in Jenkins, I can't find why, This was a try... I'll revert the change.

File: src/test/scala/com/redis/Bench.scala (hunk 1 above)
Comment (alessandrosimi-sa) [reply src/test/scala/com/redis/Bench.scala:39]: The point is the previous implementation was like this, no timeout. Does anybody know what the Banch class suppose to do?

**Pull request merged**